depscan/cli.py

```diff
@@ -744,6 +744,22 @@ def main():
         pkg_vulnerabilities,
         pkg_group_rows,
     ) = (None, None, None, None, None, None)
+    if os.getenv("GITHUB_ACTIONS").lower() == "__appthreat_dep-scan-action" \
+        and not os.getenv("INPUT_THANK_YOU") == ("I have sponsored "
+                                                 "OWASP-dep-scan."):
+        console.print(
+            Panel(
+                "You have not indicated that you have sponsored "
+                "OWASP-dep-scan.\n\nDonate at: "
+                "https://owasp.org/donate/?reponame=www-project-dep-scan"
```

Comment:
Use link syntax

https://rich.readthedocs.io/en/stable/markup.html#links

Reply:
@prabhu Couldn't get this to work even locally.

Did `[link=https://owasp.org/donate/?reponame=www-project-dep-scan&title=OWASP+depscan]project page[/link]`

![image](https://github.com/owasp-dep-scan/dep-scan/assets/80227828/169081b8-5226-461a-ad8f-0a0bd6089d48)

I tried just putting page instead of project page in case that was messing something up, but it still didn't work...